packages/tooltip/src/Tooltip/Tooltip.tsx

```diff
@@ -111,6 +113,7 @@ function Tooltip({
   const setOpen =
     isControlled && controlledSetOpen ? controlledSetOpen : uncontrolledSetOpen;
 
+  const timeoutRef = useRef<NodeJS.Timeout | null>(null);
```

Comment:
Should we also clear this in a useEffect?

Reply:
I guess in rare cases where a user hovers over the trigger and the component unmounts before the user can move the mouse away from the trigger.